src/docfx/cli/Program.cs

```
@@ -22,6 +22,8 @@ internal class Program
                     await Build.Run(docset, options, log);
                     break;
             }
+
+            // TODO: try catch all unhandled exception here with error handling
```

>// TODO: try catch all unhandled exception here with error handling [](start = 12, length = 67)

since you will try catch all exception here